src/loaders/jobs/docker_img/scripts/compute_genome_entrypoint.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
-conda run -n $CONDA_ENV \
+# Check if micromamba is available
+if command -v micromamba >/dev/null 2>&1; then
+  echo "micromamba found, using micromamba"
+  conda_cmd="micromamba"
+# Check if conda is available
+elif command -v conda >/dev/null 2>&1; then
+  echo "conda found, using conda"
+  conda_cmd="conda"
+else
+  echo "Neither conda nor micromamba found. Please install either conda or micromamba."
+  exit 1
+fi
+
+$conda_cmd run -n $CONDA_ENV \
```

Comment:
I think it would be fine to have a custom entrypoint script for tools when needed rather than having one script with a bunch of `if else` blocks.

Or alternatively, make `$conda_cmd` an environment variable defaulting to `conda` and set that env var to `micromamba` in the microtrait docker file.

Reply:
👍